src/components/page-header/master-manager.tsx

```diff
@@ -0,0 +1,227 @@
+    React.useEffect(() => {
+        if (isOpen) {
+            setList(getMasterNodeTypes(graph.current));
+        }
+    }, [isOpen, refreshNodes]);
+
+    const [openImport, setOpenImport] = React.useState<string | undefined>(undefined);
+
+    const handleReplace = (s: string) => {
+        const p = JSON.parse(s);
+        const id = p.id ? p.id : p.randomId;
+        const param: MasterParam = {
+            randomId: id,
+            label: p.label ?? id,
+            nodeType: p.nodeType ?? p.type,
+            transform: p.transform ?? defaultMasterTransform,
+            svgs: p.svgs,
+            components: p.components,
+            color: p.color,
+            core: p.core,
+            version: p.version,
+        };
+        if (!param.version || param.version < 2) {
+            toast({
+                title: 'Outdated configuration!',
+                status: 'error' as const,
+                duration: 9000,
+                isClosable: true,
+            });
+            return;
+        }
+        graph.current
+            .filterNodes(
+                node =>
+                    graph.current.getNodeAttribute(node, 'type') === MiscNodeType.Master &&
+                    graph.current.getNodeAttributes(node)[MiscNodeType.Master]!.randomId === openImport
+            )
+            .forEach(node => {
+                const newParam = structuredClone(param);
+                const nodeAttrs = graph.current.getNodeAttributes(node);
+                const attrs = structuredClone(nodeAttrs[MiscNodeType.Master]!);
+
+                const getComponentValue = (query: string) => {
+                    attrs.components.forEach(c => {
+                        if (c.id === query) {
+                            return c.value ?? c.defaultValue;
+                        }
+                    });
+                    return undefined;
+                };
+
+                newParam.components.forEach((c, i) => {
+                    newParam.components[i].value = getComponentValue(c.id) ?? c.defaultValue;
+                });
+                if (newParam.color !== undefined)
+                    newParam.color.value = attrs.color ? newParam.color.value : newParam.color.defaultValue;
+                graph.current.mergeNodeAttributes(node, { [MiscNodeType.Master]: newParam });
+            });
+        dispatch(refreshNodesThunk());
+        dispatch(saveGraph(graph.current.export()));
+    };
+
+    const handleDownload = (p: MasterParam) => {
+        const param = {
+            id: p.randomId,
+            type: p.nodeType,
+            svgs: p.svgs,
+            components: p.components,
+            color: p.color,
+            core: p.core,
+            version: p.version,
+        };
+        param.components.forEach((c, i) => {
+            param.components[i].value = c.defaultValue;
+        });
+        downloadAs(`RMP_Master_Node_${new Date().valueOf()}.json`, 'application/json', JSON.stringify(param));
```

Comment:
Do we really need a download here? Users should have their master node work available in rmp-designer?

Reply:
Maybe they get a rmp work from gallery or someway, and they don't have the designer configuration

Reply:
So Gallery will provide features that allow users to import in one click?

Reply:
Users can import the downloaded file in designer.